src/openzaak/components/zaken/admin/zaken.py

```diff
@@ -111,6 +117,11 @@ class StatusAdmin(AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmin):
     viewset = "openzaak.components.zaken.api.viewsets.StatusViewSet"
     inlines = [SubStatusForStatusInline]
 
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        if not change:
+            send_zaak_cloudevent(ZAAK_GEMUTEERD, obj.zaak, request)
```

Comment:
Could you rewrite this (and the other cloud events you added) to make use of the utils provided by `notification_api_common` to send this to Open Notificaties instead (same as Floris did in https://github.com/open-zaak/open-zaak/pull/2267/files)? Let's also add a conditional to only send the cloud event if `settings.ENABLE_CLOUD_EVENTS` is True